### PR TITLE
Completed TODO task

### PR DESCRIPTION
**Pull request by shalini-tiwari**

### CODE REVIEW

File: arr=[1,2,3,4,5].vb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+arr=[1,2,3,4,5]
+
+max=arr[0]
+n=len(arr)
+
+for i in range(i,n):
```
Comment (subhayu99) [inline arr=[1,2,3,4,5].vb]: Remove the 2nd appearance of `i`

**Review by subhayu99 — CHANGES_REQUESTED** [review]
.

File: arr=[1,2,3,4,5].vb (hunk 1 above)
Comment (DeepNinja07x) [inline arr=[1,2,3,4,5].vb]:
```suggestion
for i in range(0,n=len(arr)):

```

File: arr=[1,2,3,4,5].vb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+arr=[1,2,3,4,5]
+
+max=arr[0]
+n=len(arr)
```
Comment (DeepNinja07x) [inline arr=[1,2,3,4,5].vb]: delete this syntax

    > n=len(arr)

**Review by DeepNinja07x — CHANGES_REQUESTED** [review]

**Review by subhayu99 — APPROVED** [review]

**Pull request merged**